### PR TITLE
new version number and maintainer

### PR DESCRIPTION
**Pull request by FGarridoV**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by FGarridoV** [comment]
@moorepants

**Comment by FGarridoV** [comment]
@conda-forge-admin, please rerender

**Comment by FGarridoV** [comment]
@moorepants Is there something else I should do? or I can merge it now?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ source:
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
```
Comment (moorepants) [inline recipe/meta.yaml:17]: Yes, build number should bump on every change.

**Pull request merged**